rspirv/dr/loader.rs

```diff
@@ -32,26 +32,32 @@ impl Error {
     ///
     /// This method is intended to be used by fmt::Display and error::Error to
     /// avoid duplication in implementation. So it's private.
-    fn describe(&self) -> &str {
-        match *self {
-            Error::NestedFunction => "found nested function",
-            Error::UnclosedFunction => "found unclosed function",
-            Error::MismatchedFunctionEnd => "found mismatched OpFunctionEnd",
+    fn describe(&self) -> String {
```

Comment:
We could also return `Cow<'static, str>` here to avoid allocation of strings where not necessary

Reply:
Done